examples/example-subgraph/dist/subgraph.yaml

```diff
@@ -16,7 +16,7 @@ dataSources:
         - ExampleEntity
       abis:
         - name: ExampleContract
-          file: ExampleSubgraph/abis/ExampleContract.json
+          file: abis/ExampleContract.json
       eventHandlers:
         - event: ExampleEvent(string)
           handler: handleExampleEvent
```

Comment:
The handler name does not match the function name in mapping.ts.

Reply:
Good catch — fixed!